### PR TITLE
Fix golint failures of pkg/util/parsers pkg/util/sysctl pkg/util/system

### PR DESCRIPTION
**Pull request by SataQiu**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:
Fix golint failures of pkg/util/parsers pkg/util/sysctl pkg/util/system

**Which issue(s) this PR fixes**:

Ref #68026

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/util/sysctl/sysctl.go [hunk 1]
```diff
@@ -24,25 +24,41 @@ import (
...
 const (
-	sysctlBase         = "/proc/sys"
-	VmOvercommitMemory = "vm/overcommit_memory"
-	VmPanicOnOOM       = "vm/panic_on_oom"
-	KernelPanic        = "kernel/panic"
-	KernelPanicOnOops  = "kernel/panic_on_oops"
-	RootMaxKeys        = "kernel/keys/root_maxkeys"
-	RootMaxBytes       = "kernel/keys/root_maxbytes"
+	sysctlBase = "/proc/sys"
+	// VMOvercommitMemory defines the memory over-commit policy used by kernel.
```
Comment (mattjmcnaughton) [inline pkg/util/sysctl/sysctl.go]: Nit: Imho, For many of these, the comment makes it appear as if the constant defines the actual value of the variable, as opposed to the name of the variable.

For example, for `VMOvercommitMemory` could we say `VMOvercommitMemory refers to the sysctl variable responsible for defining the memory over-commit policy used by kernel`.

**Review by mattjmcnaughton — APPROVED** [review]
/lgtm

Thank you for fixing these :)

**Comment by mattjmcnaughton** [comment]
/lgtm

thanks for addressing my nit :)

**Review by fejta — COMMENTED** [review]
/lgtm
/approve

**Review by fejta — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**